recipes/pymablock/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pymablock" %}
+{% set version = "1.0.0.post0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version.split('.post')[0] }}
```

Comment:
One way around this might be to add a dummy output which published `version: "{{ version }}.post0"` which simply installed the `1.0.0` package and then nuke the dummy output after version `1.0.0`/`1.0.0.post0` is published in the feedstock

Reply:
An example of using a dummy output to publish both hypen and underscore names for a package is:
https://github.com/conda-forge/staged-recipes/pull/22712/files#diff-52aa6814802d1de4fdc96694db80edea2881939572e5ae3faf649f2d3e13ce14R35-R45

Reply:
I'm not 100% sure that'll work (I've never tried publishing multiple versions of the same package) but might be worthwhile giving a go.

Reply:
All that said, I'm fine with the package being published as-is.

A dummy version could always be added later, whenever it became an issue.

I don't have merge rights here though - just doing a drive-by review.

Reply:
Thanks! I think using this a post0 is the easiest. I've removed the special treatment of `.post0` version.